docs/reference/indices/simulate-template.asciidoc

```diff
@@ -0,0 +1,308 @@
+[[indices-simulate-template]]
+=== Simulate index template API
+++++
+<titleabbrev>Simulate template</titleabbrev>
+++++
+
+experimental[]
+
+Returns the index settings that would be applied by a particular 
+<<indices-templates, index template>>.
+
+////
+[source,console]
+--------------------------------------------------
+PUT _component_template/component_template1
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        }
+      }
+    }
+  }
+}
+
+PUT _component_template/other_component_template
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "ip_address": {
+          "type": "ip"
+        }
+      }
+    }
+  }
+}
+
+PUT _index_template/template_1
+{
+  "index_patterns": ["te*", "bar*"],
+  "template": {
+    "settings": {
+      "number_of_shards": 1
+    },
+    "mappings": {
+      "_source": {
+        "enabled": false
+      },
+      "properties": {
+        "host_name": {
+          "type": "keyword"
+        },
+        "created_at": {
+          "type": "date",
+          "format": "EEE MMM dd HH:mm:ss Z yyyy"
+        }
+      }
+    },
+    "aliases": {
+      "mydata": { }
+    }
+  },
+  "priority": 10,
+  "composed_of": ["component_template1", "other_component_template"],
+  "version": 3,
+  "_meta": {
+    "description": "my custom"
+  }
+}
+--------------------------------------------------
+// TESTSETUP
```

Comment:
Not a big deal, but you could probably simplify this setup to just include the index template.

Seems like a lot of setup that isn't ever surfaced to the user.